File: problems/problems_1847/solution.ts
function closestRoom(rooms: number[][], queries: number[][]): number[] {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const rooms: number[][] = JSON.parse(inputValues[0]);
	const queries: number[][] = JSON.parse(inputValues[1]);
	return closestRoom(rooms, queries);
}

File: problems/problems_1847/solution.go
package problem1847

import (
	"encoding/json"
	"log"
	"strings"
)

func closestRoom(rooms [][]int, queries [][]int) []int {
    
}

func Solve(inputJsonValues string) interface{} {
	inputValues := strings.Split(inputJsonValues, "\n")
	var rooms [][]int
	var queries [][]int

	if err := json.Unmarshal([]byte(inputValues[0]), &rooms); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &queries); err != nil {
		log.Fatal(err)
	}

	return closestRoom(rooms, queries)
}
